signal-hook-mio/tests/mio_0_7.rs

```diff
@@ -1,15 +1,21 @@
-#![cfg(feature = "support-v0_7")]
+#![cfg(any(feature = "support-v0_7", feature = "support-v0_8"))]
```

Comment:
I think this'll not test both features if both are enabled. Can you create a separate one for mio 0.8? Can you also check that there's a CI setup with mio 0.8 enabled (there might be some with `--all-features` or some such).